### PR TITLE
modal updated - now can open/close multiple times

### LINKED ISSUE #25: Geo markers alternate every 15 seconds on the map
How to reproduce
------------------------
1. Bring up a development environment: [Frontend instructions](https://github.com/OSS-team-zulu/Zulu#running-fe-locally-for-development) & [Backend instructions](https://github.com/OSS-team-zulu/Zulu#running-be-locally-for-development)
2. Load test data points by running at `./backend`
```shell
PYTHONPATH=$PYTHONPATH:$PWD python contrib/load_points.py
```
3. Open browser at `http://localhost:3000/`
4. Wait for 15 seconds, see the cities geo-points disappear, after 15 seconds they come back (keeps alternating)

See video at: https://drive.google.com/file/d/1zizAQ3-U2dm5HJQ_Nu9xANfgxH9nW9yk/view?usp=sharing

Additional information
------------------------------
This seems like a ReactJS rendering info, see this code:
https://github.com/OSS-team-zulu/Zulu/blob/4b39d0a6118d43c272ac0fa191b08233b7b8b675/frontend/zulu/src/components/Map/MapGeojsonMarkers.jsx#L64-L74

In  `componentDidMount` we set an interval to take the user location from the browser every 15 seconds - this seems to cause them to alternate - we're probably not doing something right in the interval logic.

### LINKED ISSUE #40: BUG: Post story popup can't be opened again after closing
The post modal can't be reopened in a session, requiring a reload.

Probably an issue with lifetime in react since we aren't exactly react pros just yet.

### PR DESCRIPTION
**Pull request by ornokrean**

### CODE REVIEW

**Comment by nvgoldin** [comment]
@ornokrean - I merged https://github.com/OSS-team-zulu/Zulu/pull/41 yesterday - just noticed that branch had some of the changes from here (and now this one is conflicting). Can you update what is the status of this one? thx

File: frontend/zulu/src/components/Map/MapGeojsonMarkers.jsx [hunk 1]
```diff
@@ -1,224 +1,231 @@
...
+              color="primary"
+              aria-label="edit"
+              tooltip="Add a new story!"
+              onClick={this.showModal}>
+              <FaPlus />
+            </Fab>
+          </Container>
+        </div>
+        {
+          <Modal position={[this.state.lat, this.state.lng]} show={this.state.showModal} onHide={this.closeModal.bind(this)}>
```
Comment (jherskow) [inline frontend/zulu/src/components/Map/MapGeojsonMarkers.jsx:175]: If only we knew react 😅

Great work!

**Review by jherskow — APPROVED** [review]
Excellent! thanks very much, we've been hunting this bug for ages.

the design looks great as well 💯

**Comment by nvgoldin** [comment]
Thanks @ornokrean - LGTM

**Pull request merged**